modules/ROOT/pages/_partials/partner-and-customer-guide.adoc

```diff
@@ -46,11 +46,11 @@ Before upgrading and testing your custom connector for Java 17, you must be fami
 
 * Target compilation level
 +
-NOTE: The maximum supported version for running your applications is Java 17 for Mule runtime 4.6.x and later. During the packaging of your app, all code, including third-party dependencies, *must* be compiled for Java 8.
+In Mule 4.9.0 and later, you can compile any Mule project for Java 17 or lower. Additional dependencies must be compiled for Java 17 or lower. 
 +
-Because you must compile your custom connector with Java 8, use the Java compiler configuration from the Java SDK parent POM. You can’t use language features from Java 11 or Java 17. 
+The `mule-modules-parent` should be upgraded to version 1.9.0 or later. The connector's minimum Mule version will also be 4.9.0, ensuring compatibility only with Mule 4.9.0 or later. See xref:java-version-support.adoc#bytecode_level[Bytecode Level Configuration].
```

Comment:
> The `mule-modules-parent` should be upgraded to version 1.9.0 or later

Only when compiling for Java > 8 (or using 3pp libraries compiled like that) is necessary. If staying in Java 8 is a possiblity, then this is not necessary nor a recommendation.